include/usb/usb_common.h

```diff
@@ -101,9 +101,11 @@
  * D4...0:Reserved -> 0
  */
 #define USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP  0x20
-#define USB_CONFIGURATION_ATTRIBUTES                0xC0       \
-	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP,        \
-	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0)))
+#define USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED   0xC0
+#define USB_CONFIGURATION_ATTRIBUTES ((COND_CODE_1(USB_SELF_POWERED, \
+	   (USB_CONFIGURATION_ATTRIBUTES_SELF_POWERED), (0)))   \
+	| (COND_CODE_1(CONFIG_USB_DEVICE_REMOTE_WAKEUP,         \
+	   (USB_CONFIGURATION_ATTRIBUTES_REMOTE_WAKEUP), (0))))
```

Comment:
(0) does not make any sense

Reply:
That was in the original code, so I left it and emulated it, but I agree it makes no sense.